systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -148,23 +149,23 @@ protected void changeKafkaAndLogFormatVersion(CommonVersionModificationData vers
         // #######################################################################
         String examplesPath = "";
         if (versionModificationData.getToUrl().equals("HEAD")) {
-            examplesPath = PATH_TO_PACKAGING_EXAMPLES + "";
+            examplesPath = PATH_TO_PACKAGING;
         } else {
             File dir = FileUtils.downloadAndUnzip(versionModificationData.getToUrl());
-            examplesPath = dir.getAbsolutePath() + "/" + versionModificationData.getToExamples() + "/examples";
+            examplesPath = dir.getAbsolutePath() + "/" + versionModificationData.getToExamples();
         }
 
-        kafkaYaml = new File(examplesPath + "/kafka/kafka-persistent.yaml");
+        kafkaYaml = new File(examplesPath + versionModificationData.getKafkaFilePathAfter());
         LOGGER.info("Deploying Kafka from: {}", kafkaYaml.getPath());
         // Change kafka version of it's empty (null is for remove the version)
         String defaultValueForVersions = kafkaVersionFromCR == null ? null : TestKafkaVersion.getSpecificVersion(kafkaVersionFromCR).messageVersion();
         cmdKubeClient().applyContent(KafkaUtils.changeOrRemoveKafkaConfiguration(kafkaYaml, kafkaVersionFromCR, defaultValueForVersions, defaultValueForVersions));
 
-        kafkaUserYaml = new File(examplesPath + "/user/kafka-user.yaml");
+        kafkaUserYaml = new File(examplesPath + "/examples/user/kafka-user.yaml");
```

Comment:
why not use `PATH_TO_PACKAGING_EXAMPLES` ?

Reply:
Because it's not always packaging from where we are getting the examples

+ the path for Kafka files is changed now and there can be anything inside the packaging folder. Because I'm changing for Kafka, I need to change it everywhere else.

Reply:
Makes sense. Maybe use constant for `examples` as well and use it in other constants, but maybe it is not needed.

Reply:
Maybe it would make sense to have these file paths in one of the classes that collects info for the scenario and add getters, which would then just return the exact paths - it would be better than having it hardcoded on X places.